source/material_model/compositing.cc

```diff
@@ -86,16 +86,31 @@ namespace aspect
     {
       typename Interface<dim>::MaterialModelOutputs base_output(out.viscosities.size(),
                                                                 this->introspection().n_compositional_fields);
+      create_additional_named_outputs(base_output);
```

Comment:
Please leave a comment here that this is an incomplete fix, as it can only create the additional outputs generated by material models, but not by other places.